src/rest/v0/block/mod.rs

```diff
@@ -0,0 +1,41 @@
+pub mod next_id;
+
+use crate::rest::server_service::PathPredicate;
+use actix_web::error::{Error as ActixError, ErrorBadRequest, ErrorInternalServerError};
+use actix_web::{App, Path, State};
+use blockcfg::mock::Mockchain;
+use blockchain::BlockchainR;
+use bytes::Bytes;
+use chain_core::property::Serialize;
+use chain_storage::store::BlockStore;
+
+pub fn create_handler(
+    blockchain: BlockchainR<Mockchain>,
+) -> impl Fn(&str) -> App<BlockchainR<Mockchain>> + Send + Sync + Clone + 'static {
+    move |prefix: &str| {
+        let full_prefix = format!("{}/v0/block", prefix);
+
+        App::with_state(blockchain.clone())
+            .filter(PathPredicate::new(&full_prefix, "[^/]+"))
```

Comment:
It's better to use [raw string literals][raw-string-literal] for regexp literals.

[raw-string-literal]: https://doc.rust-lang.org/reference/tokens.html#raw-string-literals